src/Commands/Workflow/Info/LogsCommand.php

```diff
@@ -35,7 +35,7 @@ function ($op) {
 
         if (empty($operations)) {
             $this->log()->notice('Workflow does not contain any operations.');
-        } else if (empty($log_operations)) {
+        } elseif (empty($log_operations)) {
```

Comment:
`elseif | else if ` is this the same |  is there a difference?

Reply:
They are functionally the same. The difference is that `elseif` is preferred by the PSR-2 standard but only emits a warning when violated, so until now we've never seen the linter complain of it while the warnings were off.